rmc/src/lib.rs

```diff
@@ -545,4 +552,68 @@ mod tests {
             assert_eq!(multisignatures[0].as_signable(), &hash);
         }
     }
+
+    #[tokio::test]
+    async fn scheduler_yields_proper_order_of_tasks() {
+        let mut scheduler = DoublingDelayScheduler::new(Duration::from_millis(100));
```

Comment:
[Just a free comment] How about dividing all the delays by 10? These two tests last ~500ms, which is not a big deal, but I guess we could have  50ms as well with zero cost :stuck_out_tongue: